Makefile

```diff
@@ -182,16 +186,40 @@ docker-push: ## Push docker image with the manager.
 ##@ Deployment
 
 install: manifests kustomize install-authorino ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	@if [ $(NAMESPACE) != '' ];then \
+		echo "Setting Custom Namespace: $(NAMESPACE)"; \
+		cd $(PROJECT_DIR)/config/install && $(KUSTOMIZE) edit set namespace $(NAMESPACE); \
```

Comment:
Do you have the diff of what this target yields with a custom namespace vs the default one? Likely, this is only because of the role bindings created for the operator, so the it sets the right ServiceAccount, correct?

Reply:
I'll create a diff

Reply:
The diff is in the gist. I comment out then clean up code when creating the diff.
https://gist.github.com/Boomatang/becfda6a1659f7e864ac039a439889bd